### ISSUE TITLE
Events not triggering

### ISSUE THREAD

**Issue by apfelbox** [Splidejs/react-splide#31]
### Checks

- [X] Not a duplicate.
- [X] Not a question, feature request, or anything other than a bug report directly related to React Splide. Use Discussions for these topics: https://github.com/Splidejs/splide/discussions

### Version

0.5.3

### Description

I have the following code:

```jsx
<Splide
    onMove={() => console.log("move")}
    onMoved={() => console.log("moved")}
    onMounted={() => console.log("mounted")}
    onActive={() => console.log("active")}
>
    {/* ... */}
</Splide>
```

And `active` is triggered once at the beginning, as well as `mounted`.
`move` and `moved` is triggered never.

### Steps to Reproduce

1. Use the implementation above to reproduce
2.
...

### Expected Behaviour

All events should be triggered.

**Comment by NaotoshiFujita** [comment]
It looks working in 0.5.3.
https://codesandbox.io/s/event-8jus5

I've also tested the latest version and found no bug.

**Comment by apfelbox** [comment]
Hi @NaotoshiFujita,

thanks for the response + your example.

I found the issue: the events don't work anymore, as soon as you sync two sliders, see here:
https://codesandbox.io/s/event-forked-e6vsm

**Comment by NaotoshiFujita** [comment]
Thank you for the additional info. I've fixed the bug(^0.6.8).

**Comment by apfelbox** [comment]
I just noticed I never actually saved my example, sorry about that.

Thanks so much for the update!

I just updated the example to the new version, but now the sync is not working properly anymore?

**Comment by apfelbox** [comment]
In `0.6.7` the transitions are properly working, but the events are not triggered.
In `0.6.8` the events are triggered, but the images don't change (just the dots)

**Comment by NaotoshiFujita** [comment]
Okay. I will check it again 😥

**Comment by NaotoshiFujita** [comment]
Fixed in 0.6.9!

**Comment by apfelbox** [comment]
Looks good, thank you so much @NaotoshiFujita ! 🙌
